### ISSUE TITLE
[Question] How to ensure return types

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#391]
Hello there. I have a question for you: How can I ensure return types in my application with PHPCS + Slevomat? For example:
```php
function testing(): int { // passes, as :int is a valid PHP 7 return type
    return 1;
}

/**
 * @return int  // passes, as int is a valid return type annotation
 */
function testing() {
    return 1;
}

function testing() { // fails, as there're missing both annotation or return type
    return 1; 
}
```

Thank you for your help :smile:

**Comment by Majkl578** [comment]
Heh, [`SlevomatCodingStandard.TypeHints.TypeHintDeclaration`](https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintstypehintdeclaration-) does this, right? It's one of the core sniffs which Doctrine is also using.
2nd declaration is invalid.
Or am I missing something?

**Comment by carusogabriel** [comment]
> 2nd declaration is invalid.

@Majkl578 I'm migrating a legacy code base for PHP 7, and I'd like to introduce types. Sometimes we can't ensure with PHP 7, but at least if we could provide any return documentation, I'd be useful!

**Comment by kukulich** [comment]
The sniff will report second and third function. The function with return annotation is fixable - annotation can be converted to return type. The function without annotation and type hint will only be reported.
